### ISSUE TITLE
Newdecls doesn't trigger warning for using old enum structs

### ISSUE THREAD

**Issue by SlidyBat** [alliedmodders/sourcepawn#294]
Using `#pragma newdecls required` should trigger warnings for using old syntax. Now that there is a new syntax available for enum structs I think the old syntax should also trigger a warning.

**Comment by dvander** [comment]
I agree, but I'm conflicted because it could suddenly cause a ton of warnings on plugins. This feature is only ~2 days old. It might be better to wait until 1.11.

**Comment by SlidyBat** [comment]
Yeah that's fair enough, I just wasn't sure if this was something intentional or not. :)
